dvc/command/ls/__init__.py

```diff
@@ -1,28 +1,43 @@
 import argparse
 import logging
+import sys
 
 from dvc.command.base import append_doc_link
 from dvc.command.base import CmdBaseNoRepo
+from dvc.command.ls.ls_colors import LsColors
 from dvc.exceptions import DvcException
 
 
 logger = logging.getLogger(__name__)
 
 
+def _prettify(entries, with_color=False):
+    if with_color:
+        ls_colors = LsColors()
+        fmt = ls_colors.format
+    else:
+
+        def fmt(entry):
+            return entry["path"]
+
+    return [fmt(entry) for entry in entries]
+
+
```

Comment:
```suggestion
def _prettify(entries, with_color=False):
    def fmt(entry):
        return entry["path"]
    if with_color:
        ls_colors = LsColors()
        fmt = ls_colors.format
    return [fmt(entry) for entry in entries]
```

Or, maybe, even use a lambda?

Reply:
I've remembered why I did it as it is
with lambda there is an [E371](https://www.flake8rules.com/rules/E731) style error
```python
def _prettify(entries, with_color=False):
    if with_color:
        ls_colors = LsColors()
        fmt = ls_colors.format
    else:
        fmt = lambda entry: entry["path"]

    return [fmt(entry) for entry in entries]
```
```
dvc/command/ls/__init__.py:17:5: E731 do not assign a lambda expression, use a def
```
Unfortunately with you suggestion I am getting [F811](https://www.flake8rules.com/rules/F811.html)
```python
def _prettify(entries, with_color=False):
     def fmt(entry):
        return entry["path"]

    if with_color:
        ls_colors = LsColors()
        fmt = ls_colors.format
    return [fmt(entry) for entry in entries]
```
```
dvc/command/ls/__init__.py:20:9: F811 redefinition of unused 'fmt' from line 15
```

Reply:
@JIoJIaJIu, It's fine to ignore redefinition warnings (it's what I suggested). But, I'm fine as-is, it's only a suggestion.